src/components/button/Button.js

```diff
@@ -42,6 +42,9 @@ export class ButtonComponent extends BaseComponent {
       this.label = this.text(this.component.label);
       this.element.appendChild(this.label);
     }
+    if (this.component.customScript) {
+        this.customScript = this.text(this.component.customScript);
```

Comment:
Why is this needed?  I don't see that this does anything other than create a DOM element that has the contents of the script. I believe this would still work without this script.